charts/kube-prometheus-stack/hack/sync_grafana_dashboards.py

```diff
@@ -27,18 +27,27 @@ def new_representer(dumper, data):
     return new_representer
 
 
+refs = {
+    # https://github.com/prometheus-operator/kube-prometheus
+    'kube-prometheus': 'a8ba97a150c75be42010c75d10b720c55e182f1a',
+    # https://github.com/kubernetes-monitoring/kubernetes-mixin
+    'kubernetes-mixin': '883f294bc636e2cd019a64328a1dbfa53edbc985',
+    # https://github.com/etcd-io/etcd
+    'etcd': '2e7ed80be246380fc50ca46aa193348a28935380',
+}
+
```

Comment:
Does this means that now we need to update this manually? Also on the other file?

Reply:
> If so, we should add that logic

Yes, otherwise the CI would randomly fail.

Renovate is designed for this, while dependabot does not have the capabilities for this.

If you would agree, I would create an additional issue to discuss about renovate. If we have renovate here, a lot of tasks could be automated.

Reply:
I'm happy to use whatever can make our life better 😄 This is probably a global discussions for the repository, since other chart owners will be able to use it too.

Still, we should add the update logic to this PR since we don't know if that discussion will take long or not. WDYT?

Reply:
https://github.com/prometheus-community/helm-charts/issues/4292

What you recommend? Should I add main branches here so we may get random CI failures? Or should I add such an script as you request?

Reply:
I would use a logic like we have for the CRDs -> we run manually to update. Maybe the file `update_crds.sh` needs to be renamed to `update_dependencies.sh` and include this new logic there.

Reply:
I still split them into to: `update_crds.sh` and `update_mixins.sh`